src/components/projects/wbs-details/project-container/project-container.tsx

```diff
@@ -42,7 +42,7 @@ const ProjectContainer: React.FC<ProjectContainerProps> = ({ wbsNum }: ProjectCo
         headerRight={<></>}
         body={
           <>
-            {data!.workPackages.map((ele: WorkPackage) => (
+            {data!.workPackages.map((ele: WorkPackageSum) => (
```

Comment:
Personally, I'd go with `WPSummary` instead, because `WP` is a common abbreviation for Work Package the application.